crates/uv-configuration/src/preview.rs

```diff
@@ -1,6 +1,7 @@
 use std::fmt::{Display, Formatter};
 
 #[derive(Debug, Clone, Copy, PartialEq, Eq, Default)]
+#[cfg_attr(feature = "clap", derive(clap::ValueEnum))]
```

Comment:
Why was this needed now?

Reply:
Good catch, i added the flag in the wrong place initially